applications/convection_diffusion_application/python_scripts/pfem2_convection_process.py

```diff
@@ -0,0 +1,85 @@
+import KratosMultiphysics as KM
```

Comment:
you are adding this file to an old folder!
The Condiff app was moved to a new folder